generic3g/ComponentSpecParser.F90

```diff
@@ -62,6 +63,7 @@ module mapl3g_ComponentSpecParser
    character(*), parameter :: KEY_UNGRIDDED_DIM_COORDINATES = 'coordinates'
    character(*), parameter :: KEY_VERTICAL_DIM_SPEC = 'vertical_dim_spec'
    character(*), parameter :: KEY_ACCUMULATION_TYPE = 'accumulation_type'
+   character(*), parameter :: KEY_RUN_DT = 'run_dt'
```

Comment:
This is fine for the PR, but we should review with Atanas for long-term "how do we want to spell it".

Reply:
Got it